scootapi/client/run_job_cmd.go

```diff
@@ -65,5 +71,9 @@ func (c *runJobCmd) run(cl *Client, cmd *cobra.Command, args []string) error {
 			return fmt.Errorf("Error running job: %v %T", err, err)
 		}
 	}
+
+	log.Println(fmt.Printf("JobID:%s\n", jobId))
```

Comment:
1) fmt.Printf already prints to stdout, so you don't need to wrap it in a log.Println
2) for stdout, I'd suggest just printing the jobId
3) then *also* log.Println with more context

That will make stdout consumable by other programs, and stderr useful for humans. WDYT?